### ISSUE TITLE
Accessibility menu via touch screen.

### ISSUE THREAD

**Issue by renpytom** [renpy/renpy#5561]
In touch mode, the accessibility menu should be accessed by a very long press (> 5s) in the center of the screen (middle square of the 3x3 grid).
